### PR TITLE
Fixes(#61): Adds backend code coverage reference

### PR DESCRIPTION
**Pull request by Nik-09**
Adds the reference link of backend code coverage in the `Debugging backend test` wiki. 
Issue link: https://github.com/oppia/oppia-web-developer-docs/issues/61

### CODE REVIEW

**Comment by Nik-09** [comment]
Hi @U8NWXD PTAL, once you are free.
Thank you.

File: Debug-backend-tests.md [hunk 1]
```diff
@@ -89,6 +90,15 @@ python -m scripts.run_backend_tests --test_path jobs
 
 **Make sure that you can reproduce the problem you are trying to debug when you run the test in isolation!** While rare, it is possible for a test failure to have been caused by previous tests. In these cases, you may not be able to reproduce the problem when you only run the test that initially failed.
 
+## Complete coverage
+
+For identifying the missing coverage lines run the below command.
```
Comment (gp201) [inline Debug-backend-tests.md]:
```suggestion
To identify the missing coverage lines, run the below command.
```

Comment (gp201) [inline Debug-backend-tests.md:95]:
```suggestion
To identify the missing coverage lines, run the below command.
```

Comment (gp201) [inline Debug-backend-tests.md:95]:
```suggestion
To identify the missing coverage lines, run the below command.
```

File: Debug-backend-tests.md [hunk 2]
```diff
@@ -89,6 +90,15 @@ python -m scripts.run_backend_tests --test_path jobs
...
+```console
+python -m scripts.run_backend_tests --generate_coverage_report
+```
+
+For more understanding about the backend code coverage kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```
Comment (gp201) [inline Debug-backend-tests.md]:
```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

Comment (gp201) [inline Debug-backend-tests.md:100]:
```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

Comment (gp201) [inline Debug-backend-tests.md:100]:
```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

Comment (gp201) [inline Debug-backend-tests.md:100]:
```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @Nik-09 ! I left some comments. PTAL

File: Debug-backend-tests.md (hunk 1 above)
Comment (Nik-09) [reply Debug-backend-tests.md]: Updated

File: Debug-backend-tests.md (hunk 2 above)
Comment (Nik-09) [reply Debug-backend-tests.md]: Updated

**Review by Nik-09 — COMMENTED** [review]
Hi, @gp201 I have addressed the comments, PTAL.
Thank you.

**Review by gp201 — APPROVED** [review]
LGTM

**Pull request merged**